src/ToDoLibrary.js

```diff
@@ -0,0 +1,15 @@
+import ToDo from './ToDo.js';
```

Comment:
- [ ] Please add a function that enables a user to edit the text of the tasks as requested in the instructions. 👍

Reply:
Done! 👍